### PR TITLE
Add performance marks to timing payload

### PR DESCRIPTION
**Pull request by ironsidevsquincy**
cc: @tom-parker

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 const Timing = require('./analytics/navigation-timing');
 const Cta = require('./analytics/cta');
-const scrollDepth = require('./analytics/scroll-depth');
```
Comment (ironsidevsquincy) [inline src/main.js]: was causing a lint error

File: bower.json [hunk 2]
```diff
@@ -16,6 +16,7 @@
   ],
   "dependencies": {
     "lodash": "https://github.com/lodash/lodash.git#3.10.1-npm",
-    "ftdomdelegate": "^2.0.3"
+    "ftdomdelegate": "^2.0.3",
+    "o-viewport": "^1.4.0"
```
Comment (ironsidevsquincy) [inline bower.json]: used by this module

**Comment by tom-parker** [comment]
Looks good, the only concern is are we going to miss all the slow font loads because the call to Spoor will have already happened? And so the data will be skewed?

**Comment by ironsidevsquincy** [comment]
Yes, could be an issue. Once we get the data, lets see how close the fontsLoaded mark is to the loadEvent. If they're close, can assume some data is being lost

**Pull request merged**